main.ts

```diff
@@ -857,30 +865,30 @@ class InlineLexer {
   };
 
   static escapes = 
-    (text?: string) => text ? text.replace(InlineLexer.rules._escapes, '$1') : text;
+    (text: string): string => text ? text.replace(InlineLexer.rules._escapes, '$1') : text;
 
-  options: MarkedOptions;
+  options: MarkedOptions | undefined;
   links: {[key: string]: Link};
   rules: Rules;
   renderer: any;
   inLink: any;
   inRawBlock: any;
 
   constructor(links: {[key: string]: Link}, options?: MarkedOptions) {
-    this.options = options || marked.defaults;
+    this.options = options;
```

Comment:
Why is `marked.defaults` dropped?

Reply:
Sorry, I am not sure how or why this happened. Moving it back does not seem to break anything, so I will do that 😄